library/Icinga/Web/Navigation/Mobile/MobileConfigMenu.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/* Icinga Web 2 | (c) 2022 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Web\Navigation\Mobile;
+
+use Icinga\Application\Icinga;
+use Icinga\Authentication\Auth;
+use Icinga\Common\HealthBadgeTrait;
+use Icinga\Web\Navigation\ConfigMenu;
+use ipl\Html\Attributes;
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\Web\Url;
+use ipl\Web\Widget\Icon;
+
+class MobileConfigMenu extends ConfigMenu
+{
+    use HealthBadgeTrait;
+
+    protected $tag = 'ul';
+
+    protected $defaultAttributes = [
+        'class' => 'nav user-nav-item',
+        'id' => 'mobile-config-menu'
+    ];
+
+    protected $flyoutID = 'mobile-config-menu-flyout';
+
+    protected function isSelectedItem($item)
```

Comment:
That's 100% the same as in the super class. Drop it please.

Reply:
✅